merged_schema/HPXMLMerged.xsd

```diff
@@ -2532,189 +2532,289 @@
 		</xs:choice>
 		<xs:attribute name="dataSource" type="DataSource"/>
 	</xs:complexType>
-	<xs:complexType name="Pools">
+	<xs:complexType name="PoolSpaType">
 		<xs:sequence>
-			<xs:element maxOccurs="unbounded" name="Pool">
+			<xs:group ref="SystemInfo"/>
+			<xs:element minOccurs="0" name="Type" type="PoolType">
+				<xs:annotation>
+					<xs:documentation>Indicates if the pool is above or below ground.</xs:documentation>
```

Comment:
Update all the references inside `PoolSpaType` from "pool" to "pool or stationary spa"?